apps/form-management-app/src/app/state/form/selectors.ts

```diff
@@ -1,4 +1,3 @@
-import { AppState } from '../store';
+// The selectors have been moved to form.slice.ts to keep them close to the state definition.
 
```

Comment:
revert these, lets keep the selectors seperated from the slice to keep it clean, so we know all the selectors are in one place